samples/sensor/distance_measure/sample.yaml

```
@@ -0,0 +1,18 @@
+sample:
+  name: Distance measurement sample
+common:
+  tags: sensor
+  harness: console
+  harness_config:
+    type: one_line
+    regex:
+      - "^[a-zA-Z0-9_,+-.]*:\\s*[0-9\\.]*\\s*\\(meters\\)$"
```

simplify + adapt to suggested reworded format string

```suggestion
      - "^[a-zA-Z0-9_,+-.]*: [0-9\\.]+m$"
```

---

Updated as per the suggestion